test_aodndata/gsla/test_gsla.py

```diff
@@ -184,9 +122,9 @@ def test_setup_upload_location_push_same_file(self):
         # create some PipelineFiles to represent the existing files on 'S3'
```

Comment:
this test was there to push the same file again. It kind of mattered because we were checking the creation date in the past. This is not required anymore, so this test should be removed/modified